src/schema.ts

```diff
@@ -40,79 +49,10 @@ export type DrumType = "bassdrum" | "hihat" | "snare";
  */
 export type Channel = DrumType | "vibraphone" | "bass";
 
-/**
- * Represents a program on the MMX's programming wheel. Used to store the programmed part of a performance.
```

Comment:
How about "The program represents the MMX's programming wheel, which can/will have multiple rotations throughout the "performance". It can also have its tempo changed or stopped completely. Other musician actions such as manually dropping marbles is done outside of the Program in the Performance."

Reply:
Sure sounds good.